src/Authentication/Authentication/Cmdlets/GetMgGraphOption.cs

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using Newtonsoft.Json.Linq;
+using System.IO;
+using System.Management.Automation;
+
+namespace Microsoft.Graph.PowerShell.Authentication.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Get, "MgGraphOption", HelpUri = "")]
+    public class GetMgGraphOption : PSCmdlet
```

Comment:
Add an `[OutputType(typeof(IGraphOption))]` attribute to the class - https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_outputtypeattribute.